README.md

```diff
@@ -0,0 +1,39 @@
+# Enumerable
+
+This project contain class MyList and a module MyEnumerable. The module MyEnumerable is implementing a subset of the functionality of [Enumerable.](https://ruby-doc.org/core-3.0.0/Enumerable.html)
+
+## Built with
+
+- Ruby
+
+## Getting Started
+
+To start using this software you need to have Ruby installed in your machine
+
```

Comment:
- [ ] For proper documentation, kindly include specific instructions on how to set up and run the project locally